modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,242 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="verify-application-pod-access-and-fpga-usagage-on-openess_{context}"]
+= Verifying application pod access and FPGA usage on OpenNESS
+
+OpenNESS is an edge computing software toolkit enabling edge platforms to onboard and manage applications and network functions on any type of network.
+
+To verify all OpenNESS features are working together, including SR-IOV binding, the device plugin, Wireless Base Band Device (bbdev) configuration, and SR-IOV (FEC) VF functionality inside a non-root pod, you can build an image and run a simple validation application for the device.
+
+For more information, go to link:http://www.openness.org[openess.org].
+
+.Prerequisites
+
+* Intel® FPGA PAC N3000 card (Optional)
+* Node or nodes installed with the n3000-operator
+* Node or nodes installed with the SR-IOV-FEC operator
+* Real-Time kernel and huge pages configured with Performance Addon Operator
+
+.Procedure
+
+. Create a namespace for the test by completing the following actions:
+
+.. Define the `test-bbdev` namespace by creating a file named `test-bbdev-namespace.yaml` file as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-bbdev
+  labels:
+    openshift.io/run-level: "1"
+----
+
+.. Create the namespace by running the following command:
```

Comment:
Excellent.  You didn't indicate to apply the CR and instead indicated what the customer accomplishes.